clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -1230,6 +1230,57 @@ parsing their headers, those should be included after the import. If the
 imported modules don't provide such a header, one can be made manually for
 improved compile time performance.
 
+Reachability of internal partition units
+----------------------------------------
+
+The internal partition units are called as implementation partition unit somewhere else.
+But the name may be confusing since implementation partition units are not implementation
+units.
+
+According to [module.reach]p1,2:
```

Comment:
It was from the draft version (https://eel.is/c++draft/module.reach#1). I changed it to the links. Hope this helps.

Reply:
Links only sort of help -- because they're live links, they get out of date very quickly. Putting a standard version number there helps the reader to understand which standard the quote was true for and they can hunt for the wording in the latest draft from there.